services/nginx/nginx.conf

```diff
@@ -18,4 +18,13 @@ server {
         proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
         proxy_set_header  X-Forwarded-Host $server_name;
     }
-}
\ No newline at end of file
+
+    location /docs {
+        proxy_pass        http://unitednepali-documentation:3000;
```

Comment:
I think you just have to add `/` after 3000, but you'll probably also have to resolve all the static file urls.

Reply:
I think you'll have to mount all the static files as defined in their `docker-compose` file on their docs. https://docusaurus.io/docs/en/docker

Reply:
nvm, you'll have to update `baseUrl` in siteConfig.js` inside docusaurus to match the forwarding endpoint of `/docs`. you'll have to update the links for all assets to match the that baseUrl unfortunately 😬 
https://github.com/facebook/Docusaurus/issues/861